pkgs/by-name/ga/gambit-project/package.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, wxGTK31
+, lib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gambit-project";
+  version = "16.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gambitproject";
+    repo = "gambit";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ElPzJDQ1q+i1OyliychSUA9pT6yGSwjn/sKV0JX5wrQ=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = [ wxGTK31 ];
+
+  meta = with lib; {
+    description = "An open-source collection of tools for doing computation in game theory";
+    homepage = "http://www.gambit-project.org";
+    license = licenses.gpl2Only;
+    maintainers = with maintainers; [ t4ccer ];
+    platforms = platforms.linux ++ platforms.darwin ++ platforms.windows;
```

Comment:
```suggestion
    platforms = platforms.unix ++ platforms.windows;
```

Reply:
Why? The website doesn't mention BSD, etc. support

Reply:
Why not? The website does not mention other Unix-likes as unsupported.
You even included Darwin despite the error on Ofborg...